mne/source_estimate.py

```diff
@@ -2252,7 +2252,7 @@ def compute_morph_matrix(subject_from, subject_to, vertices_from, vertices_to,
     elif isinstance(morpher[1], list):
         morpher = morpher[0]
     else:
-        morpher = sparse_block_diag(morpher, format='csr')
+        morpher = sparse_block_diag(morpher, fmt='csr')
```

Comment:
yes, or use nosetests. Ask for help if you need. And make sure to read contributing guidelines: http://martinos.org/mne/stable/contributing.html